packages/framework/src/Command/FixExtendedClassesAnnotationsCommand.php

```diff
@@ -0,0 +1,334 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Command;
+
+use Roave\BetterReflection\BetterReflection;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use Roave\BetterReflection\Reflection\ReflectionMethod;
+use Roave\BetterReflection\Reflection\ReflectionProperty;
+use Shopsys\FrameworkBundle\Component\ClassExtension\ClassExtensionRegistry;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+
+class FixExtendedClassesAnnotationsCommand extends Command
```

Comment:
This whole command is holding logic too many things (load/save a file, add/replace methods, add/replace parameters, replace annotations, work as a command). 
It's a violation of SRP (to be clear – I don't mind the command fixes everything, I do mind that all logic is in this class). And it's really hard to understand the 334 LoC in a single file.

It's not testable (where are the tests? 😉 ), adding another fixing rule would be quite hard.

I know it would not be easy, but please split the responsibilities and make at least some basic unit tests.
It's quite crucial functionality and I want to be somehow sure of it :)

Reply:
refactored heavily :slightly_frowning_face: